src/handler.ts

```diff
@@ -27,35 +27,36 @@ export async function handlePullRequest (context: Context): Promise<void> {
     return
   }
 
-  if(config.useReviewGroups && !config.reviewGroups){
+  if (config.useReviewGroups && !config.reviewGroups) {
     throw new Error('Error in configuration file to do with using review groups. Expected \'reviewGroups\' variable to be set because the variable \'useReviewGroups\' = true.')
   }
 
-  if(config.useAssigneeGroups && !config.assigneeGroups){
+  if (config.useAssigneeGroups && !config.assigneeGroups) {
     throw new Error('Error in configuration file to do with using review groups. Expected \'assigneeGroups\' variable to be set because the variable \'useAssigneeGroups\' = true.')
   }
 
-  const reviewers: string[] = await chooseReviewers(context, config, [], context.payload.repository.owner.login)
-  await chooseAssignees(context, config, reviewers, context.payload.repository.owner.login)
+  const reviewers: string[] = await chooseReviewers(context, config, [], context.payload.pull_request.user.login)
+  await chooseAssignees(context, config, reviewers, context.payload.pull_request.user.login)
 }
 
-
-export async function chooseReviewers(context: Context, config: AppConfig, reviewers: string[], owner: string) {
-  if(!config.reviewers && !config.reviewGroups) return []
+export async function chooseReviewers (context: Context, config: AppConfig, reviewers: string[], owner: string) {
+  if (!config.reviewers && !config.reviewGroups) return []
 
   // @ts-ignore
-  let useGroups: boolean = config.useReviewGroups && Object.keys(config.reviewGroups).length > 0
+  const useGroups: boolean = config.useReviewGroups && Object.keys(config.reviewGroups).length > 0
 
-  if(useGroups) {   
+  // TODO: Fix code based on rule: prefer-conditional-expression
+  // @see https://palantir.github.io/tslint/rules/prefer-conditional-expression/
+  if (useGroups) {
     reviewers = chooseUsersFromGroups(owner, config.reviewGroups, config.numberOfReviewers)
-  } else { 
-    reviewers = chooseUsers(owner, config.reviewers, config.numberOfReviewers)
+  } else {
+    reviewers = chooseUsers(config.reviewers, config.numberOfReviewers, owner)
   }
-  
-  if (config.addReviewers && reviewers) {
+
+  if (config.addReviewers && reviewers.length > 0) {
```

Comment:
This change fixed an issue which even if there are no reviewers, the API is executed and the test is failed